product-service-description/compliance-as-a-service/produkte/platform-linux.md

```diff
@@ -12,8 +12,33 @@ Other than that, no specific software needs to be installed (aside from the depe
 
 ## Supported packages
 
-- Both Entity Framework and SQLite queues can be launched on Linux, starting from version 1.3.3. 
-- Currently, SCU support is limited to the fiskaly and the Cryptovision packages, starting from version 1.3.3. Other SCUs (like Swissbit) will be supported soon.
+### Local storage support
+
+|                                                              | MX Linux   | Manjaro    | Linux Mint | Ubuntu     | Debian     |
+| ------------------------------------------------------------ | ---------- | ---------- | ---------- | ---------- | ---------- |
+| local mySQL Storage                                          | not tested | not tested | not tested | not tested | not tested |
+| local [SQLite-Storage](../features/SQLite-Storage.md)        | not tested | not tested | not tested | not tested | not tested |
+| local [EF-Storage](../features/EF-Storage.md)                | not tested | not tested | not tested | not tested | not tested |
+| Cloud-based [Azure-table-Storage](../features/Azure-table-Storage.md) | not tested | not tested | not tested | not tested | not tested |
```

Comment:
Let's remove this Azure Storage, it's not used by anyone and not available in production :)

Reply:
done